NagorniyAleksandr/SQLTask/README.md

```diff
@@ -0,0 +1 @@
+# JDBC
```

Comment:
Try to describe a business logic of your project 
and put instructions how to run 
e.g. step by step overview